### DISCUSSION TITLE
Change data format in history activity.

### DISCUSSION THREAD

**Discussion by kilim1983** [YetiForceCompany/YetiForceCRM#17575] [How to] [answered]
Hello

I want to know if there is an option to change data format in history activity. We want to see exacly when this happen (date for example 12 September 2024) not 19 days ago when is default option

![image](https://github.com/user-attachments/assets/83e71281-ed3b-44a5-aab0-5a472deed88a)

Regards,
Damian Krawczyk

**Comment by rskrzypczak** [comment, marked as answer]
Change the field value in the user settings
![image](https://github.com/user-attachments/assets/744206ea-4083-4ee4-9d68-3ff337ce715c)
